--- a/build.rs	2026-10-02 10:57:26
+++ b/build.rs	2026-10-02 10:57:27
@@ -6,7 +6,7 @@
 };
 
 fn main() -> Result<(), Box<dyn Error>> {
-    let manifest_dir = Path::new(env!("CARGO_MANIFEST_DIR"));
+    let manifest_dir_owned = std::path::PathBuf::from(std::env::var("CARGO_MANIFEST_DIR").unwrap()); let manifest_dir = manifest_dir_owned.as_path();
     let vendor_rbs = manifest_dir.join("vendor/rbs");
     let include = vendor_rbs.join("include");
     let c_src = vendor_rbs.join("src");
